test/parallel/test-diagnostics-channel-object-channel-pub-sub.js

```diff
@@ -37,3 +37,7 @@ channel.publish(input);
 // Should not publish after subscriber is unsubscribed
 channel.unsubscribe(subscriber);
 assert.ok(!channel.hasSubscribers);
+
+assert.throws(() => {
+  channel.subscribe(null);
+}, /ERR_INVALID_ARG_TYPE/);
```

Comment:
```suggestion
}, { code: 'ERR_INVALID_ARG_TYPE' });
```

Just a nit. Feel free to address it. I think this is the most used form in our tests.